ADAL/src/cache/ADTokenCacheItem+Internal.m

```diff
@@ -196,29 +219,13 @@ - (BOOL)fillItemWithResponse:(NSDictionary*)responseDictionary
     
     [self fillExpiration:responseDictionary];
     
-    BOOL isMRRT = ![NSString adIsStringNilOrBlank:[responseDictionary objectForKey:OAUTH2_RESOURCE]] && ![NSString adIsStringNilOrBlank:self.refreshToken];
+    BOOL isMRRT = ![NSString adIsStringNilOrBlank:self.resource] && ![NSString adIsStringNilOrBlank:self.refreshToken];
```

Comment:
Shall I move this line to line 211 and change it to be
self.resource CHANGED TO BE [responseDictionary objectForKey:OAUTH2_RESOURCE]
self.refreshToken CHANGED TO BE [responseDictionary objectForKey:OAUTH2_REFRESH_TOKEN]?